File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/actions/editor/enter/AddUnitFunctionSignatureTest.java
package org.jetbrains.plugins.scala.lang.actions.editor.enter;

import com.intellij.openapi.project.Project;
import junit.framework.Test;
import junit.framework.TestCase;
import org.jetbrains.annotations.NotNull;

public class AddUnitFunctionSignatureTest extends TestCase {
    public static Test suite() {
        return new AbstractEnterActionTestBase("/actions/editor/enter/addunit") {
            @Override
            protected void setSettings(@NotNull Project project) {
                super.setSettings(project);

                getScalaSettings(project).TYPE_ANNOTATION_UNIT_TYPE = true;
                getScalaSettings(project).ENFORCE_FUNCTIONAL_SYNTAX_FOR_UNIT = true;
            }
        };
    }
}

File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/actions/editor/enter/AddUnitFunctionSignatureTest.scala
package org.jetbrains.plugins.scala.lang.actions.editor.enter

import java.nio.file.Path

class AddUnitFunctionSignatureTest extends AbstractEnterActionTestBase {
  override protected def relativeTestDataPath: Path = Path.of("actions", "editor", "enter", "addunit")

  override protected def setUp(): Unit = {
    super.setUp()

    scalaCodeStyleSettings.TYPE_ANNOTATION_UNIT_TYPE = true
    scalaCodeStyleSettings.ENFORCE_FUNCTIONAL_SYNTAX_FOR_UNIT = true
  }
}
